src/components/PuzzleCell.tsx

```diff
@@ -21,6 +21,7 @@ export const PuzzleCell = (props: Props): JSX.Element => {
   return (
     <div className="grid-cell-wrapper">
       <div
+        contentEditable="true"
```

Comment:
?

Reply:
This causes the on-screen keyboard to appear on mobile. It's still a little wonky though. Multiple characters can be entered in a single cell from mobile. Not sure why. I'm guessing it's something about the key up event.

Reply:
Or maybe the issue is caused by the content editable. Does that allow editing the content within the div directly, or something?

Reply:
I should read what it actually does